crates/uv-python/src/managed.rs

```diff
@@ -127,9 +127,9 @@ impl ManagedPythonInstallations {
         ))
     }
 
-    /// Return the location of the cache directory for managed Python installations.
-    pub fn cache(&self) -> PathBuf {
-        self.root.join(".cache")
+    /// Return the location of the scratch directory for managed Python installations.
+    pub fn scratch(&self) -> PathBuf {
+        self.root.join(".temp")
```

Comment:
I think the rename here is now showing a warning:

```
WARN Ignoring malformed managed Python entry:
    Failed to parse Python installation key `.cache`: not enough `-`-separated values
```

Can you look for the usages? I think we exclude files that match this name somewhere, and now `.cache` exists for users but won't be skipped.